README.md

```diff
@@ -108,7 +108,7 @@ Usage: drs_download terra [OPTIONS]
 
 Options:
   -s, --silent                Display nothing.
-  -d, --destination_dir TEXT  Destination directory.  [default: /tmp/testing]
+  -d, --destination_dir TEXT  Destination directory.  [default: os.getcwd()]
```

Comment:
Many users will not know what `os.getcwd()` means.
Instead, please use:
[default: current directory]

Reply:
[Fixed](https://github.com/anvilproject/drs_downloader/blob/feature/manager/README.md?plain=1#L111)